test/code_size/hello_wasm_worker_wasm.js

```diff
@@ -1,50 +1,77 @@
-var b = Module, c = !!b.$ww, e = b.mem || new WebAssembly.Memory({
+var b = Module, c = "em-ww" == globalThis.name, e, m, x, y, z;
+
+if (c) {
+    var f;
+    onmessage = function(a) {
+        onmessage = null;
+        f = a = a.data;
+        e = a.mem;
+        g();
+        b.wasm = a.wasm;
+        h();
+        a.wasm = a.mem = 0;
+    };
+}
+
+function g() {}
+
+c || (e = b.mem || new WebAssembly.Memory({
     initial: 256,
     maximum: 256,
     shared: !0
-}), f = e.buffer, g = [], h, k = a => {
+}), g());
+
+var k = [], n = a => {
     a = a.data;
     let d = a._wsc;
-    d && h.get(d)(...a.x);
-}, l = a => {
-    g.push(a);
-}, n = {}, p = 1, q, r;
-
-c && (n[0] = this, addEventListener("message", l));
-
-WebAssembly.instantiate(b.wasm, {
-    a: {
-        b: (a, d) => {
-            let m = n[p] = new Worker(b.$wb);
-            m.postMessage({
-                $ww: p,
-                wasm: b.wasm,
-                js: b.js,
-                mem: e,
-                sb: a,
-                sz: d
-            });
-            m.onmessage = k;
-            return p++;
-        },
-        c: () => !1,
-        d: (a, d) => {
-            n[a].postMessage({
-                _wsc: d,
-                x: []
-            });
-        },
-        e: function() {
-            console.log("Hello from wasm worker!");
-        },
+    d && m.get(d)(...a.x);
+}, p = a => {
+    k.push(a);
+}, q = {}, r = 1, t = (a, d) => {
+    let l = q[r] = new Worker(b.$wb, {
+        name: "em-ww"
+    });
+    l.postMessage({
+        $ww: r,
+        wasm: b.wasm,
+        mem: e,
+        sb: a,
+        sz: d
+    });
+    l.onmessage = n;
+    return r++;
+}, u = () => !1, v = (a, d) => {
+    q[a].postMessage({
+        _wsc: d,
+        x: []
+    });
+};
+
+c && (q[0] = globalThis, addEventListener("message", p));
+
+function w() {
+    console.log("Hello from wasm worker!");
+}
+
+function h() {
+    x = {
+        b: t,
+        c: u,
+        d: v,
+        e: w,
         a: e
-    }
-}).then((a => {
-    a = a.instance.exports;
-    q = a.g;
-    r = a.i;
-    h = a.h;
-    c ? (a = b, r(a.sb, a.sz), removeEventListener("message", l), g = g.forEach(k), 
-    addEventListener("message", k)) : a.f();
-    c || q();
-}));
\ No newline at end of file
+    };
+    WebAssembly.instantiate(b.wasm, {
+        a: x
+    }).then((a => {
+        a = a.instance.exports;
+        y = a.g;
+        z = a.i;
+        m = a.h;
+        c ? (a = f, b.$ww = a.$ww, z(a.sb, a.sz), removeEventListener("message", p), k = k.forEach(n), 
+        addEventListener("message", n)) : a.f();
+        c || y();
+    }));
+}
+
+c || h();
```

Comment:
This looks larger than before, is that because of the inlined code?

Reply:
They was not needed before since `wasm_worker.js` was generated as a separate file that was not run through closure.

Now this code is part of the main output which closure sees so quotes are needed both on the sending and receiving size.

Once we do the followup of also inlining the AUDIO_WORKLET file I think we can completly remove these quotes and let closure minify these names (like we did in the pthread case) which will further reduce code size.